### PR TITLE
docs: refresh eliza's tagline, fix broken links, unify formatting for core concepts

### PR DESCRIPTION
**Pull request by wahndo**
# Relates to:

ai16z docs

# Risks

low risk

# Background

## What does this PR do?

- refresh the eliza's tagline (i thought this was a bit catchier and making agents plural is better aligned with the mutli-agent-ness of eliza)
- fix broken links
- unify formatting in the core concepts section

## What kind of change is this?

Documentation

## Why are we doing this? Any context or related work?

updating the tagline as proposed and making _agents_ plural better communicates eliza's capabilities

# Documentation changes needed?

No

# Testing

## Where should a reviewer start?

N/A

## Detailed testing steps

N/A

# Deploy Notes

N/A

## Database changes

N/A

## Deployment instructions

N/A

### CODE REVIEW

**Comment by monilpat** [comment]
LGTM outside of nit!

**Review by monilpat — CHANGES_REQUESTED** [review]
Please address the capitalization then good to go! Thanks!

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { themes as prismThemes } from "prism-react-renderer";
 
 const config = {
   title: "eliza",
-  tagline: "The flexible, scalable AI agent for everyone",
+  tagline: "flexible, scalable AI agents for everyone",
```
[the remark this answers is not in the document]
Comment (wahndo) [reply docs/docusaurus.config.js]: updated

**Review by monilpat — APPROVED** [review]
LGTM :) Thanks for getting this out!

**Pull request merged**